messaging.rst

```diff
@@ -85,27 +85,80 @@ Messages
 Locked Transfer
 -----------------
 
-Cancellable and expirable :term:`transfer`. Sent by a node when a transfer is being initiated, this message adds a new lock to the list of pending locks from the sending participant node.
+A LockedTransfer is a message used to reserve tokens for a new mediated transfer.
 
-Invariants
-^^^^^^^^^^
+Preconditions
+^^^^^^^^^^^^^
 
-Only valid if all the following hold:
+For a ``LockedTransfer`` to be considered valid the following conditions have to be true.
 
 - There is a channel which matches the given :term:`chain id`, :term:`token network` address, and :term:`channel identifier`.
 - The corresponding channel is in the open state.
 - The :term:`nonce` is increased by ``1`` in respect to the previous :term:`balance proof`
```

Comment:
- is it "with respect" or "in respect"? 
- is the last possible nonce always from a balance proof or can it also be from another locked transfer?